src/main/java/frc/robot/Constants.java

```diff
@@ -20,7 +20,7 @@
     // Change booleans to disable a subsystem in RobotContainer
     // spotless:off
     DriveBaseSubsystem(true),
-    CDSSubsystem      (true),
+    CDSSubsystem      (false),
```

Comment:
Undo

Reply:
we were trying to test and CDS was giving us an error when testing, so we should probably look into that, but I did change it back.